AmplifyPlugins/Core/AWSPluginsCore/Auth/AWSAuthSessionBehavior.swift

```diff
@@ -0,0 +1,64 @@
+//
```

Comment:
I'm not actually sure this type belongs in AWSPluginsCore -- open to moving it

Reply:
This seems like the correct place - other Auth definitions are also defined in this folder.